libnavigation-core/src/main/java/com/mapbox/navigation/core/reroute/NavigationRerouteController.kt

```diff
@@ -17,6 +19,9 @@ interface NavigationRerouteController : RerouteController {
      * Invoked whenever re-route is needed. For instance when a driver is off-route. Called just after
      * an off-route event.
      *
+     * In case you need the latest [RouteProgress] in your reroute logic,
+     * the SDK guarantees that [RouteProgressObserver.onRouteProgressChanged] is called before [reroute].
```

Comment:
Maybe we should specify that `onRouteProgressChanged` **with the corresponding off_route state** will be called before this method is invoked?

Reply:
thanks, done